src/svc_ioq.c

```diff
@@ -66,10 +66,45 @@
 #include <misc/opr.h>
 #include "svc_ioq.h"
 
+/* Queue per interface, determined per transport socket.
+ *
+ * Ideally, these would be some variant of weighted fair queuing.  Currently,
+ * assuming supplied by underlying OS.
+ *
+ * The assigned thread should have affinity for the interface.  Therefore, the
+ * first thread arriving for each interface is used for all subsequent work,
+ * until the interface is idle.  This assumes that the output interface is
+ * closely associated with the input interface.
+ *
+ * Note that this is a static fixed size list of interfaces.  In most cases,
+ * many of these entries will be unused.
+ *
+ * For efficiency, a mask is applied to the ifindex, possibly causing overlap of
+ * multiple interfaces.  The size is selected to be larger than expected number
+ * of concurrently active interfaces.  Size must be a power of 2 for mask.
+ */
+#define IOQ_IF_SIZE (16)
```

Comment:
I'm not sure this will work for a distro-wide package (such as is in Fedora).  Would it be possible to allow this to be configurable by the daemon using ntirpc?  It would be a malloc, but only one on startup.